src/pages/PullRequestPage/PullRequestPageTabs/PullRequestPageTabs.jsx

```diff
@@ -18,13 +19,16 @@ function PullRequestPageTabs() {
     indirectChangesCount,
     directChangedFilesCount,
     commitsCount,
+    isPrivateRepo,
```

Comment:
How do you feel about using `useRepoSettingsTeam` and getting the private variable from there, instead of adding it to this hook? It feels a little weird that we're getting that from a hook called `useTabsCounts` and that the only reason we're fetching it from there is cause we already had access to the repository resolver so adding the key is inconvenient. On the flip side we had to change 6 files total to make this happen, while only the change affects this file on the component side. I mainly feel the hook starts to drift from being what it's meant to do, returning counts 😬

Reply:
It's definitely drifting, I was mostly trying to avoid adding another network request and this was all ready being used in the code, that being said it's definitely a short cut. To be honest I wasnt aware of the `useRepoSettingsTeam` hook, I think it makes sense to refactor to use that one instead.